guides/source/form_helpers.md

```diff
@@ -944,51 +1057,32 @@ will create inputs like:
 ```
 
 You can also pass an `:index` option directly to helpers such as `text_field`,
-but it is usually less repetitive to specify this at the form builder level
-than on individual input fields.
+but it is usually less repetitive to specify this at the form builder level than
+on individual input fields.
 
-Speaking generally, the final input name will be a concatenation of the name
-given to `fields_for` / `form_with`, the `:index` option value, and the name of
-the attribute.
+The final input name is generally a concatenation of the name given to
+`fields_for` / `form_with`, the `:index` option value, and the name of the
+attribute (`person_address_primary_23_city` in the above example).
 
-Lastly, as a shortcut, instead of specifying an ID for `:index` (e.g.
-`index: address.id`), you can append `"[]"` to the given name. For example:
+Lastly, instead of specifying an ID for `:index` (e.g. `index: address.id`), you
+can append `"[]"` to the given name, as a shorthand. For example:
 
 ```erb
 <%= fields_for 'person[address][primary][]', address do |address_form| %>
   <%= address_form.text_field :city %>
 <% end %>
 ```
 
-produces exactly the same output as our original example.
-
-Forms to External Resources
----------------------------
-
-Rails' form helpers can also be used to build a form for posting data to an external resource. However, at times it can be necessary to set an `authenticity_token` for the resource; this can be done by passing an `authenticity_token: 'your_external_token'` parameter to the `form_with` options:
-
-```erb
-<%= form_with url: 'http://farfar.away/form', authenticity_token: 'external_token' do %>
-  Form contents
-<% end %>
-```
-
-Sometimes when submitting data to an external resource, like a payment gateway, the fields that can be used in the form are limited by an external API and it may be undesirable to generate an `authenticity_token`. To not send a token, simply pass `false` to the `:authenticity_token` option:
-
-```erb
-<%= form_with url: 'http://farfar.away/form', authenticity_token: false do %>
-  Form contents
-<% end %>
-```
+will produce exactly the same output as the original example.
 
 Building Complex Forms
 ----------------------
 
-Many apps grow beyond simple forms editing a single object. For example, when creating a `Person` you might want to allow the user to (on the same form) create multiple address records (home, work, etc.). When later editing that person the user should be able to add, remove, or amend addresses as necessary.
+As your application grows, you may need to create more complex forms, beyond editing a single object. For example, when creating a `Person` you can allow the user to create multiple `Address` records (home, work, etc.) within the same form. When later editing that person the user should be able to add, remove, or update addresses as well.
```

Comment:
'When later editing that person' feels strange. Maybe something like:
When editing that 'Person' instance later, the user should be able to add, remove, or update addresses as well.